### PR TITLE
12.0 mig website_crm_privacy_policy

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: website_crm_privacy_policy/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+* Jorge Camacho <jcamacho@trey.es>
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Antonio Espinosa
+  * Rafael Blasco
+  * Vicent Cubells
+  * Jairo Llopi
```
Comment (yajo) [inline website_crm_privacy_policy/readme/CONTRIBUTORS.rst]: "Llopis"

File: website_crm_privacy_policy/templates/contactus.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+<!-- Privacy Policy -->
+<template id="contactus_form"
+          inherit_id="website_crm.contactus_form"
+          name="Add privacy policy checkbox">
+    <xpath expr="//form/div/div/span[@id='o_website_form_result']/../.."
+           position="before">
+        <div class="form-group row form-field o_website_form_required_custom">
+            <div class="offset-lg-3 offset-md-4 col-md-8 col-lg-7">
+                <div class="checkbox">
```
Comment (yajo) [inline website_crm_privacy_policy/templates/contactus.xml]: Some classes have changed from BS3 here.

`<div class="form-input">`

File: website_crm_privacy_policy/templates/contactus.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                    <input type="checkbox"
+                        value="accepted"
+                        id="privacy_policy"
+                        name="privacy_policy"
+                        class="o_website_form_input"
```
Comment (yajo) [inline website_crm_privacy_policy/templates/contactus.xml]: class="o_website_form_input form-check-input"

File: website_crm_privacy_policy/templates/contactus.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                        required="required"/>
+                    <label class="control-label">
```
Comment (yajo) [inline website_crm_privacy_policy/templates/contactus.xml]: <label class="form-check-label">

**Review by yajo — CHANGES_REQUESTED** [review]
Some BS4 migration changes needed.

BTW I guess we will not get visual validation feedback until https://github.com/odoo/odoo/pull/28779 is merged.

File: website_crm_privacy_policy/templates/contactus.xml (hunk 4 above)
Comment (ernestotejeda) [reply website_crm_privacy_policy/templates/contactus.xml]: you are right @Yajo , Thanks

**Review by yajo — APPROVED** [review]
Please squash migration commits to merge.

**Review by chienandalu — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Comment by pedrobaeza** [comment]
I have made the squashing. Merging when finished.

**Pull request merged**